README.md

```diff
@@ -1,56 +1,56 @@
 
-The Webnative SDK empowers developers to build fully distributed web applications without needing a complex back-end. The SDK provides:
+The ODD SDK empowers developers to build fully distributed web applications without needing a complex back-end. The SDK provides:
 
-- **User accounts** via the browser's [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) or by using a blockchain wallet as a [webnative plugin](https://github.com/fission-codes/webnative-walletauth).
+- **User accounts** via the browser's [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) or by using a blockchain wallet as a [ODD plugin](https://github.com/oddsdk/odd-walletauth).
 - **Authorization** using [UCAN](https://ucan.xyz/).
-- **Encrypted file storage** via the [Webnative File System](https://guide.fission.codes/developers/webnative/file-system-wnfs) backed by [IPLD](https://ipld.io/).
+- **Encrypted file storage** via the [ODD File System](https://docs.odd.dev/developers/odd/file-system-wnfs) backed by [IPLD](https://ipld.io/).
 - **Key management** via websockets and a two-factor auth-like flow.
 
-Webnative applications work offline and store data encrypted for the user by leveraging the power of the web platform. You can read more about Webnative in Fission's [Webnative Guide](https://guide.fission.codes/developers/webnative). There's also an API reference which can be found at [webnative.fission.app](https://webnative.fission.app)
+ODD applications work offline and store data encrypted for the user by leveraging the power of the web platform. You can read more about the ODD SDK in Fission's [ODD SDK Guide](https://docs.odd.dev). There's also an API reference which can be found at [odd.fission.app](https://odd.fission.app)
 
 
 
 # Getting started
 
 ```ts
 // ESM
-import * as wn from "webnative"
+import * as odd from "@oddjs/odd"
 
 // Browser/UMD build
-const wn = globalThis.webnative
+const odd = globalThis.oddjs
 ```
 
 ## Creating a Program
 
-A Webnative program is an assembly of components that make up a distributed web application. Several of the components can be customized. _Let's stick with the default components for now, which means we'll be using the Web Crypto API._
+An ODD program is an assembly of components that make up a distributed web application. Several of the components can be customized. _Let's stick with the default components for now, which means we'll be using the Web Crypto API._
 
 ```ts
-const program = await wn.program({
+const program = await odd.program({
   // Can also be a string, used as an identifier for caches.
   // If you're developing multiple apps on the same localhost port,
   // make sure these differ.
   namespace: { creator: "Nullsoft", name: "Winamp" }
 
 }).catch(error => {
   switch (error) {
-    case webnative.ProgramError.InsecureContext:
-      // Webnative requires HTTPS
+    case odd.ProgramError.InsecureContext:
+      // ODD requires HTTPS
```

Comment:
When 'ODD' is used on its own (eg. not ODD Program), should we say ODD.js or ODD SDK instead?

Reply:
Yeah, good point. I think the distinction here is ODD SDK when used as a noun, and ODD when used as an adjective. I'll update the noun spots. 👌

Reply:
nice, I've been using `ODD SDK` for those cases in the new odd.dev site too 👍🏼